src/test/mochitest/browser_dbg-call-stack.js

```diff
@@ -107,8 +107,6 @@ add_task(async function() {
 
   await waitForPaused(dbg);
   const $group = findElementWithSelector(dbg, ".frames .frames-group");
-  is($group.querySelector(".title").textContent,
-    "<anonymous>", "Group has expected frame title");
   is($group.querySelector(".badge").textContent, "2", "Group has expected badge");
   is($group.querySelector(".location").textContent, "Angular",
```

Comment:
this still fails, because .location was renamed in favor of .group-description-name

```suggestion
  is($group.querySelector(".group-description-name").textContent, "Angular",
```